tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/AspNetCoreBlockMiddlewareIntegration.cs

```diff
@@ -0,0 +1,109 @@
+﻿// <copyright file="AspNetCoreBlockMiddlewareIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Extensions.DependencyInjection;
+#endif
```

Comment:
Should we make this _whole_ class .NET Framework only 🤔 

_Technically_ ASP.NET Core 2.x runs on .NET Framework, but we don't support it, and I don't know if we ever will, so can probably just exclude this entirely?